### PR TITLE
[FIX] List page spacing 

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

#180935241

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
Screenshot 2022-01-20 at 16 40 05

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/InvestigationsCard/useStyles.js
Comment (gertie-sheshe) [inline src/components/InvestigationsCard/useStyles.js:40]: Are the boxShadow changes meant to be on this PR? @nerdlet 😅

Comment (nerdlet) [reply src/components/InvestigationsCard/useStyles.js:40]: @gertie-sheshe Yikes, pushed this in the wrong branch. i'll revert it

File: src/components/Lists/useStyles.js
Comment (gertie-sheshe) [inline src/components/Lists/useStyles.js:14]: Seems to be 80px from the designs?

Screenshot 2022-01-19 at 23 19 30

File: src/components/Lists/index.js
Comment (kilemensi) [inline src/components/Lists/index.js:101]:
```suggestion
        <Typography variant="h2" className={classes.title}>
```

File: src/components/Lists/index.js
Comment (kilemensi) [inline src/components/Lists/index.js:102]: Add a story in PT to move all of these hard-coded strings to props & CMS.

**Review by kilemensi — COMMENTED** [review]
👍🏽 

--

Still can't run TrollTracker locally so don't wait for my approval on this. My only question is why are you adding spaces in two places; padding-bottom on title and margin-top on create button? On of these should be enough since both of these should be below the title.

**Review by KhadijaMahanga — COMMENTED** [review]
Looks Good!

I agree with @kilemensi on the `padding-bottom` & `margin-top` on two subsequent element. 

Also for smaller screen the `Create New List` does not align with the download actions
Screenshot 2022-01-20 at 12 44 38

File: src/components/ContentActions/index.js
Comment (kilemensi) [inline src/components/ContentActions/index.js:59]:
```suggestion
          <Grid container item md={4} alignItems="center">
```
1. We _want_ this to take the whole width in small screens,
2. Looks like you're changing this part of the code from 2 different PRs; this and #69 [here](https://github.com/CodeForAfrica/TwoopsTrackerUI/pull/69/files#diff-2071c343700538b0f64649995af238b5baf56a9e79e88e505f5de65c5b6b0d02L58-R84). My advice would be make this change in one of the PRs.

**Review by kilemensi — COMMENTED** [review]
👍🏽 

---

For everything under desktop (`md` and down), remove the left padding on file Download.

S

**Comment by kilemensi** [comment]
merge conflicts @nerdlet

File: src/components/AccountsList/useStyles.js
Comment (kilemensi) [inline src/components/AccountsList/useStyles.js:10]: Aren't we supposed to add spacing to around `title`? Why are we adding it to `section`?

**Review by kilemensi — COMMENTED** [review]
👍🏽

File: src/components/AccountsList/useStyles.js
Comment (nerdlet) [reply src/components/AccountsList/useStyles.js]: @kilemensi I think it works without any spacing both on the title or section. Removed i
Screenshot 2022-01-28 at 15 51 44
t 👇🏽

**Review by kilemensi — COMMENTED** [review]
🚀

File: src/components/ContentActions/index.js
Comment (KhadijaMahanga) [inline src/components/ContentActions/index.js:17]: @nerdlet not sure why we are adding `padding zero` here, and there is `paddingButton`

**Review by KhadijaMahanga — APPROVED** [review]
🚀

File: package.json
Comment (nerdlet) [reply package.json:113]: @kilemensi Hahaha, blame genderGap 😞

Comment (kilemensi) [reply package.json:113]: May be you should use [`nvm`](https://github.com/nvm-sh/nvm)?

File: src/components/ContentActions/index.js
Comment (nerdlet) [reply src/components/ContentActions/index.js:17]: @KhadijaMahanga Um, I think I removed this...let me confirm.

File: package.json
Comment (nerdlet) [reply package.json:113]: > May be you should use [`nvm`](https://github.com/nvm-sh/nvm)?

@kilemensi  💃🏽 This is definitely the tool I need at the moment. Thanks!

File: src/components/ContentActions/index.js
Comment (nerdlet) [reply src/components/ContentActions/index.js:17]: @KhadijaMahanga Yap removed it.

**Pull request merged**
